src/App.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import { BrowserRouter as Router, Route, Link, Redirect} from 'react-router-dom'
+import Transactions from './components/Transactions'
+import Transaction from './components/Transaction'
+import Operations from './components/Operations'
+import Breakdown from './components/Breakdown'
+import Home from './components/Home'
+import axios from 'axios'
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      transactions: [],
+    }
+  }
+
+  async componentDidMount() {
+    const transactions = await this.getTransactions()
+    // console.log(transactions.data);
+    this.setState({
+      transactions: transactions.data })
+  }
+
+  getTransactions = () => {
+    return axios.get('http://localhost:2700/transactions')
+  }
+
+
+
+  printTrans = () => {
+    console.log(this.state.transactions)
+  }
+
+  getBalance = () => {
+    let sum = 0
+    this.state.transactions.forEach(a => {sum += a.amount})
```

Comment:
Check the reduce method:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce